mantle/cmd/kola/options.go

```diff
@@ -177,14 +177,14 @@ func syncOptionsImpl(useCosa bool) error {
 		if kola.QEMUIsoOptions.IsoPath != "" {
 			kolaPlatform = "qemu-iso"
 		} else {
-			kolaPlatform = "qemu-unpriv"
+			kolaPlatform = "qemu"
 		}
 	}
 
 	// There used to be a "privileged" qemu path, it is no longer supported.
```

Comment:
Maybe:

// There used to be two qemu paths, but now there is only one.
// Alias qemu-unpriv to qemu for backwards compat.